.circleci/config.yml

```diff
@@ -9,18 +9,29 @@ test_build_filters: &test_build_filters
     only: /.*/
   tags:
     ignore: /v[0-9]+\.[0-9]+\.[0-9]+/
-jobs:
-  test:
+executors:
+  releaser:
+    docker:
+      - image: circleci/golang:1.12
+  publisher:
```

Comment:
Nit: Feels a little funny to be running tests on the `publisher`. Just confirming my assumption, it wouldn't be the same machine reused between the `test` and `publish` steps right?

Reply:
Renamed